scripts/gen_pascalvoc_fpndetector_max78002.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+DEVICE="MAX78002"
+TARGET="sdk/Examples/$DEVICE/CNN"
+COMMON_ARGS="--device $DEVICE --timer 0 --display-checkpoint --verbose"
+
```

Comment:
I suggest a comment here that clarifies this will generate code without the KAT.  Even echoing to the terminal may be a good idea.

```bash
echo "Synthesizing PASCAL-VOC without Known Answer Test (KAT)"
echo "To synthesize with KAT, see the other scripts next to this one"
echo ""
```

or something similar